Makefile

```
@@ -145,7 +145,7 @@ fixtures: \
 	curl -X PUT $(ES_URL)/$(ES_INDEX)/_settings \
 		-H 'Content-Type: application/json' \
 		-d '{"index": {"number_of_replicas": 0}}'
-	zcat data/statements.jsonl.gz | \
+	gzcat data/statements.jsonl.gz | \
```

Have you tested this silly workaround instead? :sweat_smile: 

```suggestion
	zcat < data/statements.jsonl.gz | \
```

---

I haven't chosen this solution based on the [Vy32 own answer ](https://serverfault.com/a/570026) and prefer a more [recent one](https://serverfault.com/a/952618). I could definitely go for it.

> Apparently my answer is wrong. Linux zcat will read .zip and .Z files. The issue here isn't one of stdin vs. named file, it's that the Linux maintainers of the programs modified gzcat to read a large number of unrelated file types.

---

Got it ! I update it